storage/innobase/handler/ha_innodb.cc

```diff
@@ -16114,6 +16134,13 @@ ha_innobase::external_lock(
 		DBUG_RETURN(0);
 	} else {
 		DEBUG_SYNC_C("ha_innobase_end_statement");
+		export_vars.rows.mutex.wr_lock();
+		export_vars.rows.deleted += rows.deleted;
+		export_vars.rows.inserted += rows.inserted;
+		export_vars.rows.read += rows.read;
+		export_vars.rows.updated += rows.updated;
+		export_vars.rows.mutex.wr_unlock();
+		new (&rows) typeof(rows) {};
```

Comment:
MSVC did not like the `typeof`, so I guess we have to use a named `struct` type for `rows`. The constructor call is for resetting the local `rows` counters after they have been added to the global ones.

Reply:
can `rows= {};` work out?